src/pug/blog/recap_year_2.pug

```diff
@@ -0,0 +1,97 @@
+
+  h4.bullet Many wireless prototype iterations
+  p We created a total of:
+  ul
+    li #[em 10 RF module prototypes] (later renamed as Marmota module).
+    li #[em 2 Alpakka carrier board prototypes] compatible with such RF modules.
+    li #[em 4 breakout boards] to perform hardware integration tests.
+
+  h4.bullet Alpakka 1.0 announcement and pre-order
+  p All these efforts on the wireless development eventually crystallized into the the #[+a('alpakka_v1') Alpakka 1.0 reference design], sadly we were not able to have it ready for the holiday season, since we are still putting the finishing touches.
+
+  hr
+  h3 Things that could have gone better
+
+  p There is an elephant in the room here: The development of the wireless Alpakka, in which we hit many research dead-ends, and it consumed most of our time during the whole year.
+
+  p In early 2024 we put a lot of effort trying to make the #[em Raspberry Pico W] to meet our requirements, the upgrade made a lot of sense since the #[em Alpakka v0] uses the normal (non-W) #[em Pico]. The first laboratory-condition tests were very promising, but later end-to-end testing revealed serious issues in both in the BT and WLAN approaches. After submitting bug reports to the appropriate channels, we could do nothing else but abandoning the idea of using the Pico W.
+
+  p In May 2024 we started experimenting with our own antenna solution, using a #[em Nordic NRF-24] as transceiver and a chip antenna. Again, the preliminary tests were very promising, but later designs with more complex PCBs had diminished antenna performance. We put a lot of effort trying to fix it and failing, and even with the support from experts in our community we were not able to make it work, and gave up on the chip antenna approach.
+
+  p In retrospective, we should have abandoned the chip antenna approach much earlier, but at least not all the R&D was wasted, as we learnt how to create our own #[em Raspberry RP2040] board with battery power management, which we can use in further developments.
+
+  p In October 2024 we started testing a #[em Espressif ESP8684-MINI-1] as a communication module (while still using the #[em RP2040] as main MCU). This is a somewhat convoluted solution, but still better than the alternatives. End-to-end prototypes were very satisfactory, and we were confident that this approach is the best one to move forward.
+
+  p Looking back, is easy to see why it took us so much time to figure out the wireless Alpakka, we didn't develop just one solution, but almost three, it just happened that the first two failed in the later stage of development. We could blame back luck, our lack of resources, or that we are not electrical engineers, but ultimately is hard not to feel that we failed a bit.
+
+  hr
+  h3 Things we hope to achieve in the 3rd year of Input Labs / 2025
+  h4.bullet Primary goals:
+  ul: ul
+    li Complete the #[em Alpakka 1.0 reference design], and start shipping pre-orders.
```

Comment:
We announced it already, so design is done. You could write that we are still doing final testing, but otherwise it's all about shipping the pre-orders.